tools/cell_census_builder/tests/test_builder.py

```diff
@@ -51,17 +52,18 @@ def setup_class(cls) -> None:
     def teardown_class(cls) -> None:
         cls.td.cleanup()
 
-    @dataclass
+    @attrs.define
```

Comment:
as these are immutable, suggest you use @attrs.frozen

Reply:
done